### ISSUE TITLE
How to delete folder not set to 'Calendar Items'

### ISSUE THREAD

**Issue by jmfoleyjr** [aluxnimm/outlookcaldavsynchronizer#260]
I annoyingly keep making this mistake: when creating a new Synchronization Profile and configuring the Outlook Folder, I often forget to set the "Folder Contains" to 'Calendar Items'. Then after clicking OK, etc. I get the error message, "Wrong Item Type in folder 'whatever'. It should be calendar, task or contact folder." I cannot then delete the profile and re-add with folder name 'whatever' and the correct Item Type. I get a file already exists message. So, I have to give it some alternate name, but the mis-typed folder stays out there. How to I delete this bogus folder?

Related: Why does CalDavSynchronizer choose "Mail and Post Items" as the default, and completely incorrect, Item Type? Why not pre-select the drop down with "Calendar Items" to begin with? I'm sure that's what most CalDavSynchronizer users want to select in the first place. Sure, maybe also "Contacts", but apparently NEVER "Mail and Post Items."

**Comment by aluxnimm** [comment]
Please don't double post the same issues on github and SourceForge!

Delete a folder just in the folder tree hierarchy in the Outlook navigation pane on the left.

You are right, would be better if the correct folder type is preselected. But this is a default Outlook dialog not from our add-in, which lacks the possibility ti specify the type when calling from C#.

**Comment by jmfoleyjr** [comment]
Regarding deleting the folder. Yes, the calendars do show up in the folder tree hierarchy, if the .pst is set to default. However, I had originally added the CalDavSynchronizer calendars to the "Internet Calendars.pst" which, as other posts informed me, cannot be set to default. I therefore re-added these calendars to a newly created .pst which I could then set as default. However, I did not first delete them from the "Internet Calendars" folder and, after setting the new calendar .pst as default, the "Internet Calendars" no longer appeared in the folder hierarchy. I solved this simply (and I should have thought of it before), by File > Open'ing the "Internet Calendars.pst" within Outlook whereupon the calendars do appear in the hierarchy. Problem solved.

With respect to the double-posting on Github and SourceForge, I posted on one or the other first (I forget which), but then had second thoughts that it was the right place, so I posted on the other as well. For my future benefit (and perhaps others), are particular posts more pertinent to one forum or the other? I'm thinking "how to" and usage type questions and discussions are appropriate for SourceForge and trouble-ticket/possible bugs more appropriate for github. Is something like that how it should work or is content between the forums arbitrary and just don't post the same thing to both?

**Comment by mtraeumner** [comment]
I get the same error message as @jmfoleyjr, but for me the entry isn't shown at the folder tree.
For replicability my steps, how I created it:
At Synchronization Profiles I started to create a new folder, clicking at the three points and then at the button "new". At the next window I changed the folder to calendar, but missed to changed the setting for "Folder contains", so it has the value "E-Mail..."  For additional information I#ve no pst file, I've an ost file, because I use IMAP in Outlook 2013. For me the wrong created folder isn't shown at the folder list, I can only see it if I try to create a new one at the sync profiles.
